nightingale_dispatcher/src/nightingale_dispatcher/task.py

```diff
@@ -1,6 +1,23 @@
 #!/usr/bin/env python3
+import rospy
+from nightingale_ros_bridge import BridgeConfig, RobotStatus, UserInputs
+from nightingale_msgs.srv import InterfaceCall
 
 
 class Task:
-    ERROR = -1
```

Comment:
Moving these from static variables to member variables could break things elsewhere. I remember using `Task.ERROR` as a comparison value in the mission planner PR. With this change, a Task instance would need to be constructed in order to use the constants `Task().ERROR`, which seems unnecessary